nmrfx-analyst-gui/src/main/java/org/nmrfx/analyst/gui/AnalystApp.java

```diff
@@ -77,7 +77,6 @@ public class AnalystApp extends MainApp {
     private static ViewMenuItems viewMenuActions;
     private static boolean startInAdvanced = true;
     private static boolean advancedIsActive = false;
-    private static MultipletController multipletController;
```

Comment:
This was the only reference to MultipletController, which seems to be an old version of MultipletTool as it had a lot of duplicated code, so removed the file.